src/openrct2/interface/Window.h

```diff
@@ -78,18 +78,45 @@ namespace WIDGET_FLAGS
     const WidgetFlags TOOLTIP_IS_STRING = 1 << 4;
 } // namespace WIDGET_FLAGS
 
+enum class WindowWidgetType : uint8_t
```

Comment:
Any reason for moving this over to `Window.h`? It's a widget thing, I think it should be kept there. If you need it here for the `rct_widget`, can't you just forward declare it as:
```c++
enum class WindowWidgetType : uint8_t;
```

And leave the definition there?